types/leaflet/index.d.ts

```diff
@@ -549,12 +549,16 @@ export namespace tileLayer {
     function wms(baseUrl: string, options?: WMSOptions): TileLayer.WMS;
 }
 
+export type CrossOrigin = boolean | string;
```

Comment:
Can you apply the CrossOrigin type also to the other occurrences?

Reply:
Yep, no problem. The only other place I could find it in the Leaflet documentation was in `TileLayer`:
https://leafletjs.com/reference-1.4.0.html#tilelayer-crossorigin

I've updated that interface now.

Reply:
@atd-schubert Does this PR look okay now?